geomstats/datasets/sample_sdp_2d.py

```diff
@@ -0,0 +1,131 @@
+"""Helper data classes for the MDM illustration example on SPD matrices."""
+
+import geomstats.backend as gs
+from geomstats.geometry.spd_matrices import SPDMatrices
+from geomstats.geometry.special_orthogonal import SpecialOrthogonal
+
+
+class DatasetSPD_2D():
+    """
+    Sample 2D SPD dataset.
```

Comment:
We chose to have the one-line summary directly after the """ for our docstring convention, see:
https://geomstats.github.io/contributing.html#documentation

Reply:
Need parameter and attributes descriptions in the docstring of the class, see eg:
https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

Reply:
Done.